apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,1129 @@
+        graphene.NonNull(MedianScoreOfGeoAreaAndAffectedGroupDateType)
+    )
+    median_score_by_sector_and_affected_group = graphene.List(graphene.NonNull(MedianScoreOfSectorAndAffectedGroup))
+    median_score_by_sector_and_affected_group_by_month = graphene.List(graphene.NonNull(MedianScoreOfSectorAndAffectedGroup))
+
+    @staticmethod
+    def custom_resolver(root, info, _filter):
+        assessment_qs = (
+            AssessmentRegistry.objects.filter(
+                project=info.context.active_project,
+                **get_global_filters(_filter),
+            )
+        )
+        assessment_qs_filter = AssessmentDashboardFilterSet(queryset=assessment_qs, data=_filter.get("assessment")).qs
+        methodology_attribute_qs = MethodologyAttribute.objects.select_related("assessment_registry").filter(
+            assessment_registry__in=assessment_qs_filter
+        )
+        cache_key = CacheHelper.generate_hash({
+            'project': info.context.active_project.id,
+            'filter': _filter.__dict__,
+        })
+        return AssessmentDashboardStat(
+            cache_key=cache_key,
+            assessment_registry_qs=assessment_qs_filter,
+            methodology_attribute_qs=methodology_attribute_qs,
+        )
+
+    @staticmethod
+    @node_cache(CacheKey.AssessmentDashboard.TOTAL_ASSESSMENT_COUNT)
+    def resolve_total_assessment(root: AssessmentDashboardStat, info) -> int:
+        return root.assessment_registry_qs.count()
+
+    @staticmethod
+    def resolve_total_stakeholder(root: AssessmentDashboardStat, info) -> int:
+        qs = root.assessment_registry_qs.values("stakeholders").distinct()
+        return qs.count()
+
+    @staticmethod
+    def resolve_total_collection_technique(root: AssessmentDashboardStat, info) -> int:
+        return root.methodology_attribute_qs.values("data_collection_technique").distinct().count()
+
+    @staticmethod
+    def resolve_assessment_count(root: AssessmentDashboardStat, info):
+        return (
+            root.assessment_registry_qs.values("coordinated_joint")
+            .annotate(count=Count("coordinated_joint"))
+            .order_by("coordinated_joint")
+        )
+
+    @staticmethod
+    def resolve_stakeholder_count(root: AssessmentDashboardStat, info):
+        stakeholder_counts = defaultdict(int)
+        organization_type_fields = ["stakeholders__organization_type__title"]
+
+        for field in organization_type_fields:
+            stakeholders = root.assessment_registry_qs.values(field)
+            for stakeholder in stakeholders:
+                if organization_type_title := stakeholder.get(field):
+                    stakeholder_counts[organization_type_title] += 1
+        return [
+            StakeholderCountType(
+                stakeholder=org_type_title,
+                count=count,
+            )
+            for org_type_title, count in stakeholder_counts.items()
+        ]
```

Comment:
Can you revisit this, we now only have a single table. Let's optimize this query.